test/support/foreman_tasks/task.rb

```diff
@@ -0,0 +1,47 @@
+module Support
+  module ForemanTasks
+    module Task
+      def stub_tasks!
```

Comment:
Seems like a copy-paste from https://github.com/Katello/katello/blob/master/test/support/foreman_tasks/task.rb, probably right time to extract this to foreman-tasks, to be reusable across projects.